tests/test_mock_controller.py

```diff
@@ -1,10 +1,15 @@
 import asyncio
 import asynctest
+import logging
 import status_assert_util as sau
 import yaml
 
 from lsst.ts import Dome
 
+logging.basicConfig(
+    format="%(asctime)s:%(levelname)s:%(name)s:%(message)s", level=logging.INFO
+)
```

Comment:
If you get tired of writing this you could provide a function that does this, preferably one that takes the level as an argument.

Reply:
Thanks, I'll remember that.